assets/src/dashboard/components/button/index.js

```diff
@@ -145,6 +145,7 @@ const Button = ({
     <StyledButtonByType
       as={isLink ? 'a' : 'button'}
       disabled={isDisabled}
+      onClick={(e) => e.stopPropagation()} // this is here so that links stacked on containers that have click handlers don't bubble. if an onClick is present as a prop it'll override this with ...rest
```

Comment:
I'm not familiar with the context here, but would it be bad to have a case where there's an `onClick` passed as a prop here  that doesn't stop propagation & there's also a link `onClick` bubbling up?

Would the approach just be to stop propagation in the `onClick` you pass down if that's the desired behavior? Or do you never want this to be able to allow for propagation under any circumstance?